### ISSUE TITLE
Classification metrics section: examples of input types

### ISSUE THREAD

**Issue by hieuchi911** [Lightning-AI/metrics#1318]
Documentation file: https://github.com/Lightning-AI/metrics/blob/v0.10.2/docs/source/pages/classification.rst
For binary inputs target, the example used was: ```binary_target = torch.tensor([1, 0, 2])```, while for binary I believe there should be no more than 2 different integers. So it should be ```binary_target = torch.tensor([1, 0, 1])```

**Comment by SkafteNicki** [comment]
Page was completly removed in this PR https://github.com/Lightning-AI/metrics/pull/1251 since the information was somewhat outdated. Instead the specifics of the expected shape are stated in the docs for each classification metric.
See for example:
https://torchmetrics.readthedocs.io/en/latest/classification/accuracy.html#binaryaccuracy
Closing issue.
